.gitignore

```diff
@@ -5,3 +5,4 @@
 /pkg/
 /spec/reports/
 /tmp/
+/.idea/
```

Comment:
Could you keep this list in alphabetical order?

Reply:
In this pattern, it is after html or after .bundle ?

Reply:
`/.bundle/`:

```ruby
pp %w(
/.bundle/
/Gemfile.lock
/coverage/
/html/
/pkg/
/spec/reports/
/tmp/
/.idea/
).sort
```

```text
["/.bundle/",
 "/.idea/",
 "/Gemfile.lock",
 "/coverage/",
 "/html/",
 "/pkg/",
 "/spec/reports/",
 "/tmp/"]
```